app/operations/beta/project_board_operations.rb

```diff
@@ -5,18 +5,20 @@ def self.call(*args)
         new(*args).run
       end
 
-      def initialize(id, current_user, current_flow: nil, projects_scope: Project)
-        @project_id = id
+      def initialize(current_user, project, current_flow: nil)
         @current_user = current_user
+        @project = project
         @current_flow = current_flow
-        @projects_scope = projects_scope
       end
 
       def run
-        users = project.users
+        error = { successful?: false, error: ActiveRecord::RecordNotFound.new }
```

Comment:
What do you think about creating a custom error class instead of instantiating `RecordNotFound` yourself? It'd be good to avoid instantiating Rails internal errors like that.